extensions/binding/org.eclipse.smarthome.binding.mqttbroker/src/main/java/org/eclipse/smarthome/binding/mqttbroker/internal/MqttWillAndTestamentBuilder.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.mqttbroker.internal;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.smarthome.io.transport.mqtt.MqttWillAndTestament;
+
+/**
+ * Builds a {@link MqttWillAndTestament}.
+ *
+ * @author David Graeff - Initial contribution
+ */
+public class MqttWillAndTestamentBuilder {
+    String topic;
```

Comment:
Test setup and mocking for private members is one side of the story however I have a different opinion on this.
The members from MqttWillAndTestamentBuilder are actually only accessed inside the class. So no need to have them package private here.